lib/global-styles.php

```diff
@@ -69,6 +69,25 @@ function gutenberg_experimental_global_styles_enqueue_assets() {
 	}
 }
 
+/**
+ * Fetches the preferences for each origin (core, theme, user)
+ * and enqueues the resulting SVGs for media filter support.
+ *
+ * TODO: Is this the right way to render the SVGs? Maybe I should render them as
+ * CSS variables so they can be included in the stylesheet?
+ */
+function gutenberg_experimental_global_styles_enqueue_svg_filters() {
+	if ( ! WP_Theme_JSON_Resolver_Gutenberg::theme_has_support() ) {
+		return;
+	}
+
+	$settings = gutenberg_get_default_block_editor_settings();
+	$all      = WP_Theme_JSON_Resolver_Gutenberg::get_merged_data( $settings );
+
+	// TODO: Should this be cached like the stylesheet?
+	echo $all->get_svgs();
+}
```

Comment:
I think I could simplify this by rendering to inline SVGs as CSS variables. That way we wouldn't need a separate render function for just the SVGs.

Reply:
Done in 5ec2efe